app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,18 @@
+<% provide(:title, t("about.name")) %>
+<% provide(:base_title, t("base.title")) %>
+<!DOCTYPE html>
+<html>
+<head>
+</head>
+<body>
+  <h1><%= t "about.name" %></h1>
+  <p>
+    <%= t "about.p1" %> <em><%= link_to t("base.p1"), "https://www.railstutorial.org/" %></em>, <%= t "about.p2" %>
+    <%= link_to t("about.p7"), "https://www.learnenough.com/" %> <%= t "about.p3" %>
+    <%= link_to t("about.p8"), "https://www.railstutorial.org/book" %> <%= t "about.p4" %>
+    <%= link_to t("about.p9"), "https://screencasts.railstutorial.org/" %> <%= t "about.p5" %>
```

Comment:
chú có thể dùng param với I18n nè để đỡ phải tạo quá nhìu biến I18n :p

Reply:
kiểu như thế này `  <%= t(".content_html", href_1: link_to(t(".content1_href"), "http://www.railstutorial.org/help"), href_2: link_to(t(".content2_href"), "http://www.railstutorial.org/book")) %>`

en.yml
content_html: "Get help on the Ruby on Rails Tutorial at %{href_1}.
        To get help on this sample app, see the <em> %{href_2} </em>."
content1_href: "Rails Tutorial help page"
content2_href: "Ruby on Rails Tutorial"